generated/proto/encodingpb/string.pb.go

```diff
@@ -14,18 +14,54 @@ var _ = proto.Marshal
 var _ = fmt.Errorf
 var _ = math.Inf
 
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
+
 type StringMeta struct {
-	Encoding    EncodingType    `protobuf:"varint,1,opt,name=encoding,proto3,enum=encodingpb.EncodingType" json:"encoding,omitempty"`
-	Compression CompressionType `protobuf:"varint,2,opt,name=compression,proto3,enum=encodingpb.CompressionType" json:"compression,omitempty"`
-	NumValues   int32           `protobuf:"varint,3,opt,name=num_values,json=numValues,proto3" json:"num_values,omitempty"`
-	MinValue    string          `protobuf:"bytes,4,opt,name=min_value,json=minValue,proto3" json:"min_value,omitempty"`
-	MaxValue    string          `protobuf:"bytes,5,opt,name=max_value,json=maxValue,proto3" json:"max_value,omitempty"`
+	Encoding             EncodingType    `protobuf:"varint,1,opt,name=encoding,proto3,enum=encodingpb.EncodingType" json:"encoding,omitempty"`
+	Compression          CompressionType `protobuf:"varint,2,opt,name=compression,proto3,enum=encodingpb.CompressionType" json:"compression,omitempty"`
+	NumValues            int32           `protobuf:"varint,3,opt,name=num_values,json=numValues,proto3" json:"num_values,omitempty"`
+	MinValue             []byte          `protobuf:"bytes,4,opt,name=min_value,json=minValue,proto3" json:"min_value,omitempty"`
+	MaxValue             []byte          `protobuf:"bytes,5,opt,name=max_value,json=maxValue,proto3" json:"max_value,omitempty"`
+	XXX_NoUnkeyedLiteral struct{}        `json:"-"`
```

Comment:
Are we using different protobuf generator? can we unify this somehow so these files don't always get updated when different people runs proto-gen?

Reply:
Maybe this is a versioning issue? I agree though. I'm using the same generators defined in the make command.